extensions/git/package.json

```diff
@@ -1967,6 +1977,11 @@
           "when": "scmProvider == git && scmResourceGroup == index",
           "group": "2_view@2"
         },
+        {
+          "command": "git.compareWithWorkspace",
```

Comment:
should the command name be `git.openWorktreeChange` ? To keep it inline with the existing `git.openChange`

Reply:
or `git.compareWorktreeChange` ? The current workspace is implied.